service/interpreter/interfaces_mock.go

```diff
@@ -0,0 +1,701 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: /Users/lwolczynski/indeedeng/iwf-server/service/interpreter/interfaces.go
+
+// Package interpreter is a generated GoMock package.
+package interpreter
+
+import (
+	context "context"
+	reflect "reflect"
+	time "time"
+
+	gomock "github.com/golang/mock/gomock"
+	iwfidl "github.com/indeedeng/iwf/gen/iwfidl"
+	service "github.com/indeedeng/iwf/service"
+)
+
+// MockActivityProvider is a mock of ActivityProvider interface.
+type MockActivityProvider struct {
```

Comment:
It’s better to name the file according so that it’s easier to find

Reply:
Is there a better way of creating this mock? I did it this way:
```
mockgen -source=service/interpreter/interfaces.go -package=interpreter -destination=service/interpreter/interfaces_mock.go 
```

Reply:
You can change the file name in the command or just rename the file in IDE. I am  not aware of other ways

Reply:
I'm confused why change the name at all. I mocked the `interface` file, so I thought `interfaces_mock` is a good name

Reply:
There are 13 mocked types in this file

Reply:
Oh you are right! My bad. I thought the file only contains the mock for activity provider which you used in the tests.